docker/login/README.md

```diff
@@ -0,0 +1,49 @@
+# docker/login
+
+Set up a before hook to log in to a Docker registry when the provided token/password
+environment variable is set in a subsequent task.
+
+Be default, the task will log in to Docker Hub. If you need to log in to a different
+registry, you can provide a `registry` parameter to this leaf.
+
+## Example
+
+```yaml
+tasks:
+  # ... `build` task omitted for brevity
+
+  - key: docker-login
+    call: docker/login 1.0.0
+    with:
+      username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
It's not a big deal but I'd rather not encourage people to put non-secret data in secrets. Can we hardcode a username here or use `vars.` instead?